src/popup/utils/utils.ts

```diff
@@ -301,3 +301,8 @@ export function isContainingNestedTx(tx: ITx): boolean {
 export function getInnerTransaction(tx: ITx): any {
   return isContainingNestedTx(tx) ? tx.tx?.tx : tx;
 }
+
+export function getFromLocalStorage(key: string, defaultValue: any = null) {
```

Comment:
In my opinion providing additional argument that is an alternative value is rather bed pattern. Without that you always can write the code this way:
```
const a = getFromLocalStorage('foo') || defaultValue;
```